.github/workflows/ci.yaml

```diff
@@ -28,4 +30,15 @@ jobs:
       - name: Build
         run: yarn build
       - name: Test
-        run: yarn test
+        run: yarn test -- --coverage
+      - name: Upload coverage
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{secrets.CODECOV_TOKEN}}
```

Comment:
getting a warning in build, maybe this token isn't set up? Not sure.

Reply:
You will see that the "test" script in package.json already has `--coverage` built into it, and I suspect that's the problem you encountered.

The workflows for this GitHub Action repository are designed for it to be able to publish its own tags, avoiding the need to do this manually. To ensure the coverage is reported similarly to other code-bases, the Coverage step should be added to the `pull-request` workflow. I've got a separate PR open for this - this one could probably be closed and the branch removed.